build_feature_tree.js

```diff
@@ -0,0 +1,143 @@
+const fs = require("fs");
+const path = require("path");
+const yaml = require("js-yaml");
+const _ = require("lodash");
```

Comment:
We use "_" for underscore everywhere else in our codebase, let's keep lodash as `lodash`, pls